src/main/java/com/hazelcast/aws/CollectionUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.hazelcast.aws;
+
+import java.util.Collection;
+
+class CollectionUtils {
+    private CollectionUtils() {
+    }
+
+    static boolean isNotEmpty(Collection<?> collection) {
```

Comment:
You use it only in `AwsConfig`, so I think you can just move this method there. No need for a separate utils class.